tensorboard/backend/json_util.py

```diff
@@ -51,9 +51,8 @@ def Cleanse(obj, encoding='utf-8'):
   Returns:
     Unicode JSON data structure.
   """
-  if isinstance(obj, int):
```

Comment:
Please revert this change. I special-cased `int` because it's the most likely branch. See also https://github.com/tensorflow/tensorboard/commit/c152887f924397db3c7c37f0e8d8346012e66ddc as well as